### PR TITLE
Facebook Login

### PR DESCRIPTION
**Pull request by mrosso10**
# Summary
* Added Facebook login with Omniauth2

# Test plan
1. Click on the "Sign in with Facebook" link in the navbar
2. If you are not logged in with a Facebook account you will be asked to
3. You should be logged in with your Facebook account

# Trello card
https://trello.com/c/2zp6q5Vt/21-integrar-el-blog-con-facebook-login

### CODE REVIEW

File: .config.travis.yml [hunk 1]
```diff
@@ -5,3 +5,5 @@ mandrill:
 omniauth:
   google_app_id: ""
   google_app_secret: ""
+  facebook_app_id: ""
```
Comment (Dinuuu) [inline .config.travis.yml]: remember what we talk about this

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -238,6 +238,7 @@
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
   config.omniauth :google_oauth2, app_configuration.omniauth["google_app_id"], app_configuration.omniauth["google_app_secret"], { }
+  config.omniauth :facebook, app_configuration.omniauth["facebook_app_id"], app_configuration.omniauth["facebook_app_secret"], { }
```
Comment (Dinuuu) [inline config/initializers/devise.rb]: remember about `AppConfiguration.for(:omniauth)`

**Comment by mrosso10** [comment]
Rebased and fixed!

**Comment by Dinuuu** [comment]
:+1:

**Pull request merged**